docs/introduction/1236-community.md

```diff
@@ -0,0 +1,39 @@
+---
+slug: /1236/community
+displayed_sidebar: "0.2"
+---
+
+# Join the Community
+
+Dagger is open source! Anyone is welcome to join the community and help shape the evolution of Dagger.
```

Comment:
`Open Source`

Reply:
I went for consistency and that sentence is taken from the bottom of https://dagger.io where "open source" is lower case. I'm fine wither way :-)